src/vgr_cairo.mli

```diff
@@ -0,0 +1,117 @@
+(*---------------------------------------------------------------------------
+   Copyright 2014 Arthur Wendling, Daniel C. Bünzli. All rights reserved.
+   Distributed under the BSD3 license, see license at the end of the file.
+   %%NAME%% release %%VERSION%%
+  ---------------------------------------------------------------------------*)
+
+(* Based on the Vgr_htmlc documentation by Daniel C. Bünzli. *)
+
+(** Vg Cairo renderer.
+
+    {b Dependencies:}
+    {ul {- {e {{:http://forge.ocamlcore.org/projects/cairo/}Cairo2 bindings
+     for OCaml}}}
+     {- {e {{:http://cairographics.org/}Cairo Graphics library}}}}
+
+    {e Release %%VERSION%% — %%MAINTAINER%% } *)
+
+(** {1:target Cairo render targets} *)
+
+val target : ?resolution:Gg.V2.t -> [< `PDF | `PNG | `PS | `SVG ] ->
+  Vg.Vgr.dst_stored Vg.Vgr.target
+(** [target resolution fmt] is a render target for rendering to the stored
+    destination given to {!Vg.Vgr.create} in the chosen format [fmt].
+    {ul
+    {- [resolution], specifies the rendering resolution in samples per
+       meters. The PDF, PS and SVG formats are measured in points by Cairo,
+       while the PNG format is in pixels. If unspecified, the default
+       conversion to points is used.}}
```

Comment:
No, I don't think so. Cairo defines the units used for each of its backend, but doesn't care about the physical size otherwise (so pixel surfaces don't have a "printed real life size" metadata..)

The other backends effectively do not need to respect the resolution since the conversion from meters to points doesn't need to be configured; it seemed simpler to just use the default value and let users tweak the ratio it if they desire. I think `Png of resolution` might be better than silently ignoring the argument if you care about enforcing the physical ratio.